### PR TITLE
Update carbonblack_edr readme.md

### PR DESCRIPTION
**Pull request by cherryleaf**
Markdown tidy up

Type of change

- Enhancement

## What does this PR do?

 Tidy up of markdown

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: packages/carbonblack_edr/_dev/build/docs/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # VMware Carbon Black EDR Integration
 
-The VMware Carbon Black EDR integration collects EDR Server and raw Endpoint events exported by [Carbon Black EDR Event Forwarder.](https://github.com/carbonblack/cb-event-forwarder) The following output methods are supported: `http`, `tcp`, `udp` and `file`.
+The VMware Carbon Black EDR integration collects EDR Server and raw Endpoint events exported by [Carbon Black EDR Event Forwarder.](https://github.com/carbonblack/cb-event-forwarder) The following output methods are supported: `http`, `tcp`, `udp` and `file`.
```
Comment (efd6) [inline packages/carbonblack_edr/_dev/build/docs/README.md:3]: Why is this change necessary?

File: packages/carbonblack_edr/changelog.yml [hunk 2]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.1.2"
```
[the remark this answers is not in the document]
Comment (GinnyGalicia) [reply packages/carbonblack_edr/changelog.yml:2]: Done

File: packages/carbonblack_edr/_dev/build/docs/README.md (hunk 1 above)
Comment (GinnyGalicia) [reply packages/carbonblack_edr/_dev/build/docs/README.md:3]: It was a way of pushing up the amended file from the local drive.
